authN-authZ/auth-apisix/apisix-chatqna-route.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright (C) 2024 Intel Corporation
```

Comment:
Instead of adding this file, could you re use the one in templates and add a new values file like values_megaservice.yaml and update the values accordingly so that the implementation will be consistent with helm charts